src/GraphQL/Utilities/AstPrinter.cs

```diff
@@ -264,6 +264,12 @@ public AstPrintVisitor()
                 c.Field(x => x.Value);
                 c.Print(f => f.Arg(x => x.Value));
             });
+
+            Config<NullValue>(c =>
+            {
+                c.Print(f => null);
```

Comment:
Yes, that matches the JS project behavior.

https://github.com/graphql/graphql-js/blob/master/src/language/printer.js#L79